### PR TITLE
Fix/setup py

### PR DESCRIPTION
**Pull request by oleksii-symon-corva-ai**

### CODE REVIEW

**Comment by jordanambra** [comment]
Is there a task associated with this? Do we actually require these version upgrades for our customers or can they still use older versions of them?

**Comment by oleksii-symon-corva-ai** [comment]
there is no task associated, I just saw that 1. versions in setup.py and requirements were different 2. packages option could be simplified

**Comment by oleksii-symon-corva-ai** [comment]
> can they still use older versions of them

I think that minimum version in `install_requires` should be 1. >= than version in requirements and 2. < than next major library version (example https://github.com/tiangolo/fastapi/blob/33be5fc8baa02bae949e1986f1821dae8f84f487/pyproject.toml#L36)

File: setup.py [hunk 1]
```diff
@@ -32,12 +33,7 @@
...
     keywords='corva, sdk',
-    packages=setuptools.find_packages(exclude=[".*", "data", "test"]),
-    install_requires=[
-        "pydantic >= 1.7.2",
-        "redis >= 3.5.3",
-        "requests >= 2.25.0",
-        "urllib3 >= 1.26.2"
-    ],
+    packages=["corva"],
+    install_requires=["pydantic >= 1.7.3", "redis >= 3.5.3", "requests >= 2.25.0"],
```
Comment (finomayato) [inline setup.py:37]: requirement ranges should be closed to not to be updated to the major version e.g. `~=`/`>=,<=`

Comment (jordanambra) [inline setup.py:37]: I thought this was easier to read with each requirement on its own line as well

File: setup.py [hunk 2]
```diff
@@ -32,12 +33,7 @@
...
     license='The Unlicense',
```
[the remark this answers is not in the document]
Comment (jordanambra) [reply setup.py:38]: Let me ask for support from the product team on this and we will follow up with a change, if necessary.

**Review by jordanambra — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (oleksii-symon-corva-ai) [reply setup.py]: added ranges

Comment (oleksii-symon-corva-ai) [reply setup.py]: fixed

**Review by jordanambra — APPROVED** [review]

**Pull request merged**